cli/Cargo.toml

```diff
@@ -26,6 +26,7 @@ rand = "0.8.5"
 itp-types = { git = "https://github.com/integritee-network/pallets.git", branch = "master" }
 itp-utils = { git = "https://github.com/integritee-network/pallets.git", branch = "master" }
 my-node-runtime = { package = "integritee-node-runtime", git = "https://github.com/integritee-network/integritee-node", branch = "master" }
+pallet-evm = { default-features = false, git = "https://github.com/integritee-network/frontier.git", branch = "polkadot-v0.9.26" }
```

Comment:
You can remove the `default-features = false`. But I believe it should be optional? E.g. only imported in `evm` feature?